admin/common_auth/models.py

```diff
@@ -49,6 +49,10 @@ class MyUser(AbstractBaseUser, PermissionsMixin):
     date_joined = models.DateTimeField(auto_now_add=True)
     confirmed = models.BooleanField(default=False)
     osf_id = models.CharField(max_length=5, blank=True)
+    desk_key = models.CharField(max_length=45, blank=True)
```

Comment:
Confirmed with Sara just now, `key` and `key_secret` are associated with the group, not the user.